installation/windows.md

```diff
@@ -5,16 +5,67 @@ title: Windows
 
 {% include base.html %}
 
-# Installation on Windows
+# openHAB on Windows
 
-The openHAB runtime comes as a platform-independent zip file.
-To install it, follow these simple steps.
-You will need to install Java if not already installed. Go to http://java.com to get it.
-Unzip the `openhab-offline-<version>.zip` to where it is intended to be running from, e.g. `C:\openhab`.
-Note that there can't be a space in the path, so it can't be in "Program Files" for example.
+The following instructions will guide you through the process of setting up openHAB 2.
 
-Start the server: Launch the runtime by executing the script `start.bat`.
+## Prerequisites
 
-Point your browser to `http://localhost:8080` and you should be looking at the openHAB portal page.
+OpenHAB requires Java 8 to run correctly, we reccomend a version of 8_101 and higher. 
+To get the latest version, visit the [java website](https://www.java.com/en/download/).
```

Comment:
I'd like to avoid triple-mentioning that. Do you agree it would be better to link to http://docs.openhab.org/installation/index.html#prerequisites instead or in addition to the first sentence?